tracing/trace_state.hh

```diff
@@ -68,7 +68,7 @@ private:
     lw_shared_ptr<one_session_records> _records;
     // Used for calculation of time passed since the beginning of a tracing
     // session till each tracing event.
-    elapsed_clock::time_point _start;
+    elapsed_clock::time_point _start = elapsed_clock::time_point::min();  // Sentinel value (== "unset")
```

Comment:
let's drop the comment and instead create a named, self-descriptive constant for the sentinel value

Reply:
For sentinel we usually use std::optional. but here I think `max` makes a better alternative, since then we can use std::min(local clock, supplied start) to choose something that works even if clocks are not synchronized, and between mixed versions.

However, please test mixed versions explicitly.